ui/src/components/view/InfoCard.vue

```diff
@@ -281,6 +281,9 @@
                 style="margin-left: -24px; margin-top: 5px;">
                 <a-icon type="api" />eth{{ index }} {{ eth.ipaddress }}
                 <router-link v-if="eth.networkname && eth.networkid" :to="{ path: '/guestnetwork/' + eth.networkid }">({{ eth.networkname }})</router-link>
+                <span v-if="eth.isdefault">
```

Comment:
@davidjumani the default I think is usually the first; more clearly seen from the nics tab.

Reply:
Instead of parenthesis can you use a tiny tag https://www.antdv.com/components/tag/

Reply:
Added tags for nics but not for networks so it maintains the theme

![Screenshot from 2021-06-08 10-06-42](https://user-images.githubusercontent.com/8244774/121123706-4ae6c800-c841-11eb-8b39-4a4af7f85083.png)
![Screenshot from 2021-06-08 10-06-53](https://user-images.githubusercontent.com/8244774/121123715-4c17f500-c841-11eb-96ca-d7b13468cde0.png)